config/routes.js

```diff
@@ -173,4 +173,8 @@ module.exports.routes = {
    "get /config/site": "SiteController.configSite",
    "get /config/user": "SiteController.configUser",
    "get /config/inbox": "SiteController.configInbox",
+
+   // System Information
+   // "get /information/object/:objID": "definition_manager/information-object",
+   "get /information/object/:objID/field/:ID": "definition_manager/information-field",
```

Comment:
Can we make this `/definition/info/... `, so it's consistent with our other `definition_manager` api routes?